src/test/libradosstriper/rados-striper.sh

```diff
@@ -78,9 +82,9 @@ function run() {
     diff -w $dir/xattrlist2 $dir/reflist || return 1    
     rados --pool rbd --striper rmxattr toyfile somexattr || return 1
     rados --pool rbd --striper getxattr toyfile somexattr >& $dir/rmxattrerror
-    grep -q 'No data available' $dir/rmxattrerror || return 1
+    grep -q "$notfoundattr" $dir/rmxattrerror || return 1
     rados --pool rbd getxattr toyfile.0000000000000000 somexattr >& $dir/rmxattrerror2
-    grep -q 'No data available' $dir/rmxattrerror2 || return 1
+    grep -q "$notfoundattr" $dir/rmxattrerror2 || return 1
```

Comment:
@tchaikov 
I always have to suppress the urge to completely rewrite all `|| return 1` rules,
because I like things to be consistent at all lines....
(But I've restrained myself here.)